breathecode/authenticate/tests/urls/tests_academy_student_id.py

```diff
@@ -482,12 +482,17 @@ def test_academy_student_id__put__user_exists_but_without_profile_academy(self):
     @patch('os.getenv', MagicMock(return_value='https://dotdotdotdotdot.dot'))
     def test_academy_student_id__put__with_data(self):
         """Test /academy/:id/member/:id"""
+        profile_academy = {
+            'first_name': self.bc.fake.first_name(),
+            'last_name': self.bc.fake.last_name(),
```

Comment:
Add a case when the ProfileAcademy is not being saved with the requires fields

Reply:
This comment has a slight problem, which is that at the moment of execution, it tries to pass through a PUT that is validating only 'crud_member' kinda requests but this test is trying to pass 'crud_student' so I changed it to 'crud_member' anyways even knowing that something may be already testing it in another folder

Reply:
`crud_student` and `crud_member` are capabilities, so you need add to the test `self.bc.database.create(capability='capability', profile_academy=1, role=1)`